EasyCommands/CommandParsers/ParameterProcessors.cs

```diff
@@ -356,7 +358,11 @@ public static class ParameterProcessorRegistry {
                             } else revisit = true;
                         }
                     }
-                    if (processed) { processorIndex = 0; continue; }
+                    if (processed) {
+                        Debug(String.Join(" ", commandParameters.Select(p => CommandParameterToString(p)).ToList()));
+                        processorIndex = 0;
```

Comment:
Let's add a script test for this. Don't bother asserting every line, just pick one.